bandit/reporters/progress_base.h

```diff
@@ -92,6 +100,18 @@ namespace bandit {
       }
 
     protected:
+      void update_test_duration() {
+        testcase_duration_ = std::chrono::high_resolution_clock::now() - testcase_start_time_point_;
+        testsuite_runtime_ += std::chrono::duration_cast<std::chrono::nanoseconds>(testcase_duration_);
+      }
+
+      std::string print_time(double t) {
```

Comment:
The name of the method does not represent what it is doing (i.e., it does not print anything).

Reply:
time_to_string?